py/mkrules.mk

```diff
@@ -70,6 +70,7 @@ $(OBJ): | $(HEADER_BUILD)/qstrdefs.generated.h $(HEADER_BUILD)/mpversion.h
 # - if anything in QSTR_GLOBAL_DEPENDENCIES is newer, then process all source files ($^)
 # - else, if list of newer prerequisites ($?) is not empty, then process just these ($?)
 # - else, process all source files ($^) [this covers "make -B" which can set $? to empty]
+# See more information about the QSTR process in py.pk (under "QSTR generation").
```

Comment:
py.pk -> py.mk

Also sounds better to say 'For more information about .. se ...' perhaps?